xlators/cluster/afr/src/afr-open.c

```diff
@@ -311,9 +325,50 @@ afr_fix_open(fd_t *fd, xlator_t *this)
     ret = loc_path(&local->loc, NULL);
     if (ret < 0)
         goto out;
+    local->cont.lk.getlk_rsp1 = GF_CALLOC(sizeof(*local->cont.lk.getlk_rsp1),
+                                          priv->child_count, gf_afr_mt_gf_lock);
 
     local->fd = fd_ref(fd);
     local->fd_ctx = fd_ctx;
+    local->strict_locks = _gf_false;
+    local->op = GF_FOP_OPEN;
```

Comment:
Changing the op type doesn't seem right.

Reply:
Removed this part as well.